src/views/Project/index.jsx

```diff
@@ -22,15 +25,58 @@ const tagReposMapping = repositories =>
   }, {});
 
 @hot(module)
-@graphql(issuesQuery, {
-  skip: props => !projects[props.match.params.project].repositories,
-  options: () => ({
-    fetchPolicy: 'network-only',
-    variables: {
-      searchQuery: '',
-    },
+@compose(
+  graphql(issuesQuery, {
+    skip: ({
+      match: {
+        params: { project },
+      },
+    }) => !projects[project].repositories,
+    name: 'github',
+    options: () => ({
+      fetchPolicy: 'network-only',
+      variables: {
+        searchQuery: '',
+      },
+      context: {
+        client: 'github',
+      },
+    }),
   }),
-})
+  graphql(bugsQuery, {
+    skip: ({
+      match: {
+        params: { project },
+      },
+    }) => !projects[project].products,
+    name: 'bugzilla',
+    options: ({
+      match: {
+        params: { project },
+      },
+    }) => ({
+      variables: {
+        search: {
+          keywords: ['good-first-bug'],
+          products: productsList(projects[project].products).length
+            ? productsList(projects[project].products)
+            : Object.keys(projects[project].products[0])[0],
+          components: productsList(projects[project].products).length
+            ? undefined
+            : Object.values(projects[project].products[0])[0],
```

Comment:
Why are we only picking the zero index element? What about the rest?

Reply:
the rest is picked on the fetchMore.

Reply:
Why not pick everything from the start?

Reply:
because the component is not unique for different products, e.g Bugzilla product has Administration component, and bugzilla.mozilla.org also has Administration component, then if it only specify Administration component in Bugzilla, when we combine the product and component, it may also return Administration component in bugzilla.mozilla.org